src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapCheckBSCToSirius.vue

```diff
@@ -688,16 +706,107 @@ export default {
+      }else{
+        siriusAddressSelectedName.value = 'ACCOUNT-' + siriusAddressSelected.value.substring(siriusAddressSelected.value.length-4,siriusAddressSelected.value.length)
+      }
+    });
+
+    const showAddressError = shallowRef(true);
+    const toggleContact = shallowRef(false)
+    watch(siriusAddressSelected,n=>{
+      if(n.length==40 || n.length==46){
+        checkRecipient()
+      }else{
+        showAddressError.value = true
+      }
+    })
+    const checkRecipient = () =>{
+    if(!walletState.currentLoggedInWallet){
+        return;
+    }
+    try {
+      let recipientAddress = Helper.createAddress(siriusAddressSelected.value);
+      let networkOk = Helper.checkAddressNetwork(recipientAddress, AppState.networkType);
+      if(!networkOk){
+        showAddressError.value = true;
+      }
+      else{
+        showAddressError.value = false;
+      }
+    } catch (error) {
+      try{
+        let namespaceId = Helper.createNamespaceId(siriusAddressSelected.value);
+        checkNamespace(namespaceId).then((address)=>{
+          siriusAddressSelected.value = address.plain();
+          showAddressError.value = false;
+        }).catch((error)=>{
+          showAddressError.value = true;
+        });
+      }
+      catch(error){
+        showAddressError.value = true;
+      }
+    }
+  }
+    const accounts = computed(()=>{
+      if(!walletState.currentLoggedInWallet){
+        return [];
+      }
+      let accounts = walletState.currentLoggedInWallet.accounts.map(
+        (acc)=>{ 
+          return { 
+            name: acc.name,
+            balance: acc.balance,
+            address: acc.address,
+            publicKey: acc.publicKey,
+            isMultisig: acc.getDirectParentMultisig().length ? true: false
+          }; 
+        });
+        
+       
+       let otherAccounts =walletState.currentLoggedInWallet.others.filter((acc)=> acc.type === "MULTISIG").map(
+        (acc)=>{ 
+          return { 
+            name: acc.name,
+            balance: acc.balance,
+            address: acc.address,
+            publicKey: acc.publicKey,
+            isMultisig: true
+          }; 
+        });
+        return accounts.concat(otherAccounts);
+      
+    });
+    const contacts = computed(() => {
+      if(!walletState.currentLoggedInWallet){
+        return [];
+      }
+      const wallet = walletState.currentLoggedInWallet;
+      var contact = [];
+      accounts.value.forEach((element) => {
+        contact.push({ 
+          value: Address.createFromRawAddress(element.address).pretty() ,
+          label: element.name + " - "+t('general.ownerAcc'),
+        });
+      });
+      if (wallet.contacts != undefined) {
+        wallet.contacts.forEach((element) => {
+          contact.push({
+            value: Address.createFromRawAddress(element.address).pretty(),
+            label: element.name + " - "+t('general.contact'),
+          });
+        });
       }
```

Comment:
check for duplication, should use contact name instead of account name for duplicated entry